sdk/core/azure_core_amqp/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corp. All Rights Reserved.
+
+# AMQP Stack for consumption by packages in the Azure SDK.
+[package]
+name = "azure_core_amqp"
+version = "0.19.0"
+description = "Rust client Azure SDK"
+readme = "README.md"
+authors = ["Microsoft Corp."]
+license = "MIT"
+repository = "https://github.com/azure/azure-sdk-for-rust"
+homepage = "https://github.com/azure/azure-sdk-for-rust"
+documentation = "https://docs.rs/azure_core_amqp"
+
+keywords = ["sdk", "azure", "rest", "iot", "cloud"]
+categories = ["api-bindings"]
+
+edition = "2021"
+
+[dependencies]
+async-std = { workspace = true }
+azure_core = { path = "../../core/azure_core" }
+time = { workspace = true }
+tracing = { workspace = true }
+url = { workspace = true }
+uuid = { workspace = true }
+fe2o3-amqp = { version = "0.11", features = [
+  "native-tls",
+  "tracing",
+  "uuid",
+], optional = true }
+fe2o3-amqp-ext = { version = "0.11", features = [], optional = true }
+fe2o3-amqp-management = { version = "0.11", optional = true }
+fe2o3-amqp-cbs = { version = "0.11", optional = true }
+fe2o3-amqp-types = { version = "0.11", optional = true }
+serde_amqp = { version = "0.11", optional = true, features = ["uuid"] }
+serde_bytes = { version = "0.11", optional = true }
+
+[build-dependencies]
+rustc_version = "0.4"
+
+[dev-dependencies]
+tracing-subscriber = { workspace = true, features = ["env-filter"] }
+
+[features]
+default = ["enable-fe2o3-amqp"]
```

Comment:
FWIW, I'm doing to remove "enable-" from the features for core. I don't see it anywhere else, and it's enough just to mention the project name of the default e.g., `reqwest` or `fe2o3`. This is what I've seen other projects do as well. Many crates, for example, mention `serde` by name as a feature. And just that: `serde`.

Reply:
Annoyingly, the feature name cannot match a crate name - cargo rejects it :(.

Reply:
Use "dep:{crate}" to work around that.